src/main/java/gg/solarmc/clans/command/commands/clans/InfoCommand.java

```diff
@@ -54,21 +69,31 @@ private void sendClanInfo(CommandSender player, Clan clan) {
         int kills = clan.currentKills();
         int assists = clan.currentAssists();
         int deaths = clan.currentDeaths();
-        // TODO
-        // final Set<ClanMember> clanMembers = clan.currentMembers();
+        String members = clan.currentMembers().stream().map(it -> getPlayerNameById(player.getServer(), it.userId())).collect(Collectors.joining(","));
 
         String[] info = {
                 "Information about " + clanName,
                 "Kills : " + kills,
                 "Assists : " + assists,
                 "Deaths : " + deaths,
                 "Ally : " + allyClanName,
-                // "Members : "
+                "Members : " + members
         };
-        // TODO : make it look good
         player.sendMessage(info);
     }
 
+    private String getPlayerNameById(Server server, int id) {
+        SolarPlayer solarPlayer = server.getDataCenter().lookupPlayer(id)
+                .exceptionally(e -> {
+                    LOGGER.error("Cannot lookup player by id", e);
+                    return Optional.empty();
+                }).getNow(Optional.empty()).orElse(null);
+
+        if (solarPlayer == null) return null;
+
+        return server.getOfflinePlayer(solarPlayer.getMcUuid()).getName();
```

Comment:
Server#getOfflinePlayer(UUID) does behave the way you think. Let me add SolarPlayer#getUsername